ts-defs/index.d.ts

```diff
@@ -1580,6 +1758,25 @@ interface TestFn {
      * Skips execution of all tests, except this one.
      */
     only: this;
+    /**
+     * Specifies the additional information for the test that can be used in reports
```

Comment:
Specifies the additional information for the test. This information can be used in reports.